libconfluo/src/atomic_multilog.cc

```diff
@@ -59,7 +59,14 @@ atomic_multilog::atomic_multilog(const std::string &name, const std::string &pat
   metadata_ = metadata_writer(path, false);
   // Load multilog data
   data_log_.init("data_log", path, s_mode);
-  rt_ = read_tail_type(path, s_mode);
+  rt_ = read_tail_type(path, s_mode, /* load = */ s_mode == storage::storage_mode::DURABLE);
```

Comment:
You already pass s_mode to the read_tail constructor -- can't you just perform the `s_mode == storage::storage_mode::DURABLE` check inside the constructor?

Reply:
i was worried about the edge case where you initialize a _new_ durable multilog and it tries to read an existing read tail file from a previous run. making this an explicit flag means you would only try loading when its passed in from the multilog load constructor.

not sure if this can actually happen or not.